.github/workflows/docs-and-linting.yml

```diff
@@ -9,7 +9,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: ['1.19', '1.20', '1.21']
+        # current Go releases plus the version in the go.mod are tested
+        go: ['1.18', '1.21', '1.22']
```

Comment:
This list has changed? should be 1.22 and two versions lower?

Reply:
It used to be two versions lower and I broke with that to have the two stable releases and the version that's in our go.mod.